cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In the TicketSerializer, you are validating the ticket. However, it would be better to move this validation logic to the model's clean method. This way, the validation would be applied not only when using serializers but also when you save the model instance elsewhere in your code.